src/content/developers/docs/accounts/index.md

```diff
@@ -41,9 +41,9 @@ Both account types have the ability to:
 
 Ethereum accounts have four fields:
 
-- `nonce` – a counter that indicates the number of transactions sent from the account. This ensures transactions are only processed once. In a contract account, this number represents the number of contracts created by the account
-- `balance` – the number of wei owned by this address. Wei is a denomination of ETH and there are 1e+18 wei per ETH.
-- `codeHash` – this hash refers to the _code_ of an account on the Ethereum virtual machine (EVM). Contract accounts have code fragments programmed in that can perform different operations. This EVM code gets executed if the account gets a message call. It cannot be changed unlike the other account fields. All such code fragments are contained in the state database under their corresponding hashes for later retrieval. This hash value is known as a codeHash. For externally owned accounts, the codeHash field is the hash of an empty string.
+- `nonce` – A counter that indicates the number of transactions sent from the account. This ensures transactions are only processed once. In a contract account, this number represents the number of contracts created by the account
+- `balance` – The number of wei owned by this address. Wei is a denomination of ETH and there are 1e+18 wei per ETH.
+- `codeHash` – For externally-owned accounts, this field is the hash of an empty string. For contract accounts, this field is the hash of the Ethereum virtual machine (EVM) _code_ that gets executed if the account receives a message call. These code fragments can perform various operations, and they are contained in the state database under their corresponding hashes for later retrieval. Unlike all other account fields, the codeHash is immutable, meaning it cannot be modified after its creation.
```

Comment:
```suggestion
- `nonce` – A counter that indicates the number of transactions sent from the account. This ensures transactions are only processed once. In a contract account, this number represents the number of contracts created by the account
- `balance` – The number of wei owned by this address. Wei is a denomination of ETH and there are 1e+18 wei per ETH.
- `codeHash` – This hash refers to the _code_ of an account on the Ethereum virtual machine (EVM). Contract accounts have code fragments programmed in that can perform different operations. This EVM code gets executed if the account gets a message call. It cannot be changed, unlike the other account fields. All such code fragments are contained in the state database under their corresponding hashes for later retrieval. This hash value is known as a codeHash. For externally owned accounts, the codeHash field is the hash of an empty string.
```

Reply:
Thanks for the suggestions on this. I've checked this over a few times and checked with others and we think this re-wording detracts from the readability a bit. I've suggested changes to the casing at the start of the sentence because this was incorrect.